src/features/partners/PartnerStats.jsx

```diff
@@ -6,15 +6,19 @@ import EnrollmentStatsCard from '../enrollments/EnrollmentStatCard';
 import CourseEnrollmentList from '../enrollments/CourseEnrollmentList';
 import MemberEnrollmentList from '../enrollments/MemberEnrollmentList';
 import MembershipProvider from '../members/MembershipProvider';
+import useCohorts from '../cohorts/useCohorts';
 import usePartner from './usePartner';
+import useOfferings from '../offerings/useOfferings';
 import StatCard from './StatCard';
 import ManagementMenu from './ManagementMenu';
 import PartnerHeading from './PartnerHeading';
 
 export default function PartnerStats() {
   const [partner, partnerSlug] = usePartner();
+  const [partnerOfferings] = useOfferings();
+  const [cohorts] = useCohorts();
 
-  const courseCount = partner?.offerings.length;
+  const courseCount = partnerOfferings?.length;
```

Comment:
Will this still provide the correct course count if the same course is made available to multiple different cohorts? I just want to be sure that we're counting total _partner_ offerings, not total _cohort_ offerings.

Reply:
`useOfferings` -> `selectOfferingsByPartnerSlug` takes all offerings, filters them by the current slug, and returns unique courses. 

But! `useCohorts` isn't scoped to the current partner, so currently cohorts for other partnerships are displaying in the filters. 😨 

`offerings` wasn't either, which is why I updated the hook to take a partner slug, but we probably need a more robust way to ensure state is scoped to the current partner.